bin/transform.py

```diff
@@ -144,6 +149,63 @@ def lock(metadata):
 
     return metadata_readable, metadata_irida
 
+def categorize(metadata):
+
+    needed_cols = [
+        "host_scientific_name", "host_common_name", "food_product",
+        "environmental_site", "environmental_material"
+    ]
+    missing_cols = [col for col in needed_cols if col not in metadata.columns]
+
+    if len(missing_cols)>0:
+        raise KeyError("Missing required fields: " + str(missing_cols))
+
+    metadata_readable = metadata.copy(deep=True)
+
+    metadata_readable["calc_source_type"] = "Unknown"
+
+    metadata_readable.loc[
+        metadata_readable["host_scientific_name"].str.contains("homo sapiens", case=False, na=False) |
+        metadata_readable["host_common_name"].str.contains("human", case=False, na=False),
```

Comment:
I wonder if we could have all these human terms as constants, have a regex that contains all of them, and then with a single check, check if the regex matches. The regex part might be a bit tricky, but the contains would be something like: `str.contains(HUMAN_REGEX, case=False, regex=True)`.

Then it would be easier to add or remove other terms if it comes up later.

Reply:
This may take some discussion with Emily and James, since I think they may be wanting this more strict than I currently have it (ie. such that it only accepts a specific value). Also, eventually it'll be dynamic, with the variable, user-inputted rules.

Reply:
Yeah - just confirmed with James. It shouldn't even be case-insensitive. I agree that it's pretty bad to have it so hardcoded, but I think this will make more sense with the rules tables